dice-game-ui-angular/src/app/game/game.component.ts

```diff
@@ -1,13 +1,334 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, HostListener } from '@angular/core';
 import * as d3 from 'd3';
 
+import { SimpleGameEngine, SimpleDice, Rect, Size } from '../../../../engine';
+
 @Component({
   selector: 'app-game',
   templateUrl: './game.component.html',
   styleUrls: ['./game.component.scss'],
 })
 export class GameComponent implements OnInit {
-  constructor() {}
+  private readonly setColor = '#D28EFF';
+  private readonly selectedColor = '#3f51b5';
+  private readonly emptyColor = 'lightblue';
+  private canvasMaxHeight = 600;
+
+  private gameEngine;
+  private simpleDice;
+
+  private points: FieldPoint[] = [];
+
+  private selecting = false;
+  private svg = null;
+  private startPoint: FieldPoint = null;
+
+  public dice1 = 0;
```

Comment:
You can omit `public` modifier, because it is used by TS by default.

Reply:
I don't agree with this simplification beacuse it's not completely obvious.

Reply:
I didn't say that you "should", only that you "can" :) If you don't want - don't do it. But if you decide to add `public` - add it everywhere, because code style should be consistent across the application and `ngOnInit` and `onResize` methods doesn't have any modifier. The same is for other classes.

Reply:
Eventually, I fixed it )